cmd/snap/cmd_connect_test.go

```diff
@@ -30,27 +30,19 @@ import (
 
 func (s *SnapSuite) TestConnectHelp(c *C) {
 	msg := `Usage:
-  snap.test [OPTIONS] connect <snap>:<plug> <snap>:<slot>
+  snap.test [OPTIONS] connect [<snap>:<plug>] [<snap>:<slot>]
```

Comment:
Awkward that the first entry is between [] although it's marked as required. Is this test running?